### ISSUE TITLE
Width is not applied correctly to elements

### ISSUE THREAD

**Issue by FabulousGee** [atk4/ui#656]
Given the following code results in an optical bug where the second field does not size correctly.
Instead of having two equal fields with 50% size, we have the first field sized 50% and the second field 25% since the width is applied twice.
```
$form = $app->layout->add(new \atk4\ui\Form());
$form->addHeader('Example');
$group = $form->addGroup();
$group->addField('trainingResult', ['width'=>'eight']);
$group->addField('trainingNote', ['width'=>'eight']);
```

This is the HTML which is getting generated
```
<div class="eight wide  field">
	<label for="_5694b51f__trainingNote_input">Example</label>

	<div id="_5694b51f__trainingNote" class="ui field eight wide action input" style="">
		<label></label>
		<div id="_5694b51f__trainingNote-ac" class="ui search selection dropdown  " style="width:100%">
			<input name="trainingNote" type="hidden" id="_5694b51f__trainingNote_input" value="">
			<i class="dropdown icon"></i>
			<div class="default text"></div>
			<input class="search" value="" tabindex="0">
			<div class="menu" tabindex="-1"></div></div>
		<div id="_5694b51f__trainingNote_button" class="ui  button" style="">Add new</div>
	</div>
</div>
```
![1](https://user-images.githubusercontent.com/30511025/53130982-e7e79f00-356b-11e9-9b1c-94226ebd3b2a.JPG)

And that's how it should look like to work correctly: (removed two classes from second div)
```
<div class="eight wide  field">
	<label for="_5694b51f__trainingNote_input">Example</label>

	<div id="_5694b51f__trainingNote" class="ui field action input" style="">
		<label></label>
		<div id="_5694b51f__trainingNote-ac" class="ui search selection dropdown  " style="width:100%">
			<input name="trainingNote" type="hidden" id="_5694b51f__trainingNote_input" value="">
			<i class="dropdown icon"></i>
			<div class="default text"></div>
			<input class="search" value="" tabindex="0">
			<div class="menu" tabindex="-1"></div></div>
		<div id="_5694b51f__trainingNote_button" class="ui  button" style="">Add new</div>
	</div>
</div>
```
![2](https://user-images.githubusercontent.com/30511025/53130981-e7e79f00-356b-11e9-8823-21ce9aeff52b.JPG)

**Comment by ibelar** [comment]
@FabulousGee - I am unable to reproduce you bug using your code above. 
This is the html resulting from it: https://jsfiddle.net/ibelar/xhm8j0w6/

**Comment by FabulousGee** [comment]
@ibelar sorry, maybe my example code was a bit short... I now tried a vanilla file and executed the following code:
`	class NoteModel extends \atk4\data\Model {
		function init() {
			parent::init();

			$this->addField('note');
		}
	}

	class testModel extends \atk4\data\Model {
		private static $enum_result = ['Yes', 'No'];

		function init() {
			parent::init();

			$this->addField('trainingResult',
				['enum' => self::$enum_result, 'caption' => 'Success']);
			$this->hasOne('trainingNote',
				[new NoteModel(), 'caption' => 'Note',
					'ui' => [
						'form' => [
							'Lookup',
							'plus' => TRUE,
						],
					]]);
		}
	}

	$ar = [];
	$db_array = new \atk4\data\Persistence_Array($ar);

	$app = new \atk4\ui\App();
	$app->initLayout('Admin');

	$form = $app->layout->add(new \atk4\ui\Form());
	$form->addHeader('Example');

	$m = new testModel($db_array);
	$form->setModel($m, false);

	$group = $form->addGroup();
	$group->addField('trainingResult', ['width'=>'eight']);
	$group->addField('trainingNote', ['width'=>'eight']);`

It results in:
https://jsfiddle.net/pdqu3goc/

I hope this is reproducing the bug for you, too. I am using Chrome, just in case...

**Comment by ibelar** [comment]
Fix in #671 

@FabulousGee - thanks for the supply code. Help a lot!

**Comment by FabulousGee** [comment]
Sure, you're welcome! Thanks for the fix!

Can confirm it's working.
